servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -71,6 +71,11 @@ quarkus.resteasy.gzip.enabled=true
 quarkus.http.auth.basic=false
 quarkus.oidc.enabled=false
 
+## quarkus kubernetes settings (https://quarkus.io/guides/deploying-to-kubernetes)
```

Comment:
do these things ahve to live here? The server, up until now, is fairly agnostic to how we run right?

Reply:
technically speaking, we don't need to have any of those k8s-specific settings here. I'll remove them